amethyst_audio/src/formats.rs

```diff
@@ -37,6 +39,8 @@ impl Format<AudioData> for OggFormat {
     }
 }
 
+impl SerializableFormat<AudioData> for OggFormat {}
```

Comment:
I think that all `impl SerializableFormat` should be done as part of `amethyst_assets::register_format!` macro.

Reply:
@Frizi done!